packages/client-sdk-nodejs/src/config/transport/storage/transport-strategy.ts

```diff
@@ -22,11 +22,11 @@ export interface StorageTransportStrategy {
 
   /**
    * Copy constructor to update the client-side timeout
-   * @param {number} clientTimeoutMillis
+   * @param {number} requestTimeoutMillis
```

Comment:
should I also undo this `clientTimeoutMillis` to `requestTimeoutMillis` renaming now that the new settings are confined to retry strategy and not transport/grpc settings?

Reply:
i think i have a slight pref for keeping this consistent with the existing configs for CacheClient etc, so yeah I'd probably change it back. thanks for catching that.